service-front/module/Application/view/application/pages/post_office/donor_details_match_check.twig

```diff
@@ -0,0 +1,48 @@
+{% include 'layout/id_check_banner.twig' with details_data %}
+<div class="govuk-width-container govuk-!-padding-top-5">
+
+        <form method="POST">
+            <h1 class="govuk-heading-xl govuk-!-width-two-thirds">Do the details match the ID document?</h1>
```

Comment:
Could you add `govuk-!-width-two-thirds` to the div above, rather than adding it to all of the children?